src/utils/truncate-string.test.ts

```diff
@@ -0,0 +1,9 @@
+import { truncateString } from './truncate-string';
```

Comment:
Nice util. We have a utility component called StringTruncator that you can give a max width and it will wrap the text with a tooltip that you can hover to view the full name. It's not perfect but I think it would be nice if we could display that information too.